app/views/companies/_company_members.html.haml

```diff
@@ -0,0 +1,17 @@
+- applications = company.accepted_applications_from_members
+
+- if applications.any?
+
+  .col-md-12.company_member
+
+    %table.table.table-hover
```

Comment:
use a CSS class name so the element (the table) can be styled appropriately  (ex:  `.member-employees`)